msgp/elsize_test.go

```diff
@@ -47,7 +46,6 @@ func TestBytespec(t *testing.T) {
 		mmap32:    {size: 5, extra: map32v, typ: MapType},
 	}
 
-	//func init() {
```

Comment:
I double checked if this was left here for a reason (and/or if this needed a "TODO"); it was introduced in https://github.com/tinylib/msgp/commit/43b487371e0cdd1a3fe1be9a7802d8bce917a1af / https://github.com/tinylib/msgp/pull/295), but neither the commit-message nor code-review shows a mention of this, so looks like it was just accidentally left behind.